smartsim/_core/control/controller.py

```diff
@@ -739,13 +812,64 @@ def _set_dbobjects(self, manifest: Manifest) -> None:
                             if db_script not in ensemble.db_scripts:
                                 set_script(db_script, client)
 
+    def start_telemetry_monitor(self, exp_dir: str, frequency: int) -> None:
+        logger.debug("Starting telemetry monitor process")
+        if not CONFIG.telemetry_enabled:
+            return
+
+        if (
+            self._telemetry_monitor is None
+            or self._telemetry_monitor.returncode is not None
+        ):
+            cmd = [
+                sys.executable,
+                "-m",
+                "smartsim._core.entrypoints.telemetrymonitor",
+                "-exp_dir",
+                exp_dir,
+                "-frequency",
+                str(frequency),
```

Comment:
do we care about this inconstent config frequency passing (vs Config.telemetry_cooldown below)

Reply:
removed frequency param & used `CONFIG.telemetry_frequency`